unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/MapRenderer.cs

```diff
@@ -252,9 +261,22 @@ void UpdateParcelHold()
             }
         }
 
+        private void OnKernelConfigChanged(KernelConfigModel current, KernelConfigModel previous)
+        {
+            Debug.Log("OnKernelConfigChanged");
+
+            validWorldRanges = current.validWorldRanges;;
```

Comment:
```suggestion
            validWorldRanges = current.validWorldRanges;
```
Remove the extra semi-colon.